src/nodes/bullets.rs

```diff
@@ -60,7 +60,6 @@ impl scene::Node for Bullets {
     }
```

Comment:
Ohhh, I forgot to delete bullets.rs :( 
But it was not compiling anyway - it was not it nodes.rs